sdk/communication/Azure.Communication.CallAutomation/samples/Sample1_StartDialog.md

```diff
@@ -0,0 +1,46 @@
+# Start Dialog
+
+This sample demonstrates how to add a Dialog Bot to a phone call and start it.
+
+To get started you will need:
+- A Communication Service Resource.  See [README][README] for prerequisites and instructions.
+- A Nuance Mix Bot. See [Nuance Mix][Nuance_Mix] for instructions
+
+## Creating a `CallAutomationClient`
+
+A CallDialog instance requires the creation of a CallAutomation client. This can be authenticated using the connection string acquired from an Azure Communication Resource in the Azure Portal
+```C#
+var connectionString = "<connection_string>"; // Find your Communication Services resource in the Azure portal
+CallAutomationClient callAutomationClient = new CallAutomationClient(connectionString);
+```
+
+## Dialog Dependencies and Creating a `CallDialog` Instance
+
+The Dialog relies on an already running call (see the [Create Call Sample][create_call_sample]) to run. With an existing call, take the CallConnectionId to create the CallDialog instance
+```C#
+var callConnection = client.GetCallConnection(@event.CallConnectionId);
+var callDialog = callConnection.GetCallDialog();
```

Comment:
Is our construct named "call dialog" in the SDK? 

Bringing the two words together is not really making it clear what it is an what it's doing. Can you walk me a bit through the reasoning for picking this name?

Reply:
CallDialog follows the convention of the rest of the Call Automation SDK - the Media client is CallMedia, Recording client is CallRecording. Their corresponding "constructor" methods attach Get as a prefix.